### DISCUSSION TITLE
Docker Mods

### DISCUSSION THREAD

**Discussion by lseufer** [themepark-dev/theme.park#153] [Q&A] [answered]
Hi,

I am successfully loading the docker mods but the TP_THEME variable doesn't change anything. No matter the theme name I enter organizr-dark is loading.

Any idea how to fix this?

Thanks!

**Comment by GilbN** [comment, marked as answer]
Does the container logs say anything?
And which mod are you trying?

**Reply by lseufer** [reply]
<img width="638" alt="Screenshot 2021-03-05 at 10 11 49" src="https://user-images.githubusercontent.com/7261297/110093790-656d5900-7d9b-11eb-8e88-951231a217d6.png">


Nothing helpful from the logs I am afraid

**Reply by GilbN** [reply]
Did you recreate the container after you added TP_THEME?
It will only sed the changes if the domain variable isnt in the html file.
https://github.com/gilbN/theme.park/blob/docker-mods/nzbget/root/etc/cont-init.d/98-themepark#L24

**Reply by lseufer** [reply]
(faceplam) Sorry, I just needed to recreate the container. It's working perfectly now. Thanks!
